EXAMPLES.md

```diff
@@ -1,7 +1,25 @@
-curl -i -XPOST localhost:8080/routes -d '{
-  "name": "THE_ROUTE",
-  "description": "this is a route",
-  "activate_at": "2015-09-28T16:58:56.957",
+curl -ik -XPOST https://innkeeper.pathfinder-staging.zalan.do./routes -d '{
+    "route": {
+      "matcher": {
+        "path_matcher": {
+          "match": "/hello-*",
```

Comment:
not quite right: when using wildcards, the wildcard should have a name. when no name is required, conventionally, we use '_' as the name. ==> "/hello-*_"